pkg/cmd/update-tls-artifacts/generate-owners/generate_owners_flags.go

```diff
@@ -48,39 +49,31 @@ func NewGenerateOwnershipCommand(streams genericclioptions.IOStreams) *cobra.Com
 
 func NewGenerateOwnersFlags(streams genericclioptions.IOStreams) *GenerateOwnersFlags {
 	return &GenerateOwnersFlags{
-		RawTLSInfoDir:       "tls/raw-data",
-		TLSOwnershipInfoDir: "tls/ownership",
-		ViolationDir:        "tls/violations/ownership",
-		IOStreams:           streams,
+		TLSInfoDir: "tls",
+		IOStreams:  streams,
 	}
 }
 
 func (f *GenerateOwnersFlags) BindFlags(flags *pflag.FlagSet) {
-	flags.StringVar(&f.RawTLSInfoDir, "raw-tls-dir", f.RawTLSInfoDir, "The directory where the raw TLS info is located.")
-	flags.StringVar(&f.TLSOwnershipInfoDir, "ownership-dir", f.TLSOwnershipInfoDir, "The directory where the  TLS ownership info is should be written.")
-	flags.StringVar(&f.ViolationDir, "violation-dir", f.ViolationDir, "The directory where the  TLS ownership info is should be written.")
+	flags.StringVar(&f.TLSInfoDir, "ownership-dir", f.TLSInfoDir, "The directory where the  TLS ownership info is should be written.")
 	flags.BoolVar(&f.Verify, "verify", f.Verify, "Verify content, don't mutate.")
 }
 
 func (f *GenerateOwnersFlags) Validate() error {
-	if len(f.RawTLSInfoDir) == 0 {
-		return fmt.Errorf("--raw-tls-dir must be specified")
-	}
-	if len(f.TLSOwnershipInfoDir) == 0 {
+	if len(f.TLSInfoDir) == 0 {
 		return fmt.Errorf("--ownership-dir must be specified")
 	}
-	if len(f.ViolationDir) == 0 {
-		return fmt.Errorf("--violation-dir must be specified")
-	}
 	return nil
 }
 
 func (f *GenerateOwnersFlags) ToOptions() (*GenerateOwnersOptions, error) {
 	return &GenerateOwnersOptions{
-		RawTLSInfoDir:       f.RawTLSInfoDir,
-		TLSOwnershipInfoDir: f.TLSOwnershipInfoDir,
-		ViolationDir:        f.ViolationDir,
-		Verify:              f.Verify,
-		IOStreams:           f.IOStreams,
+		TLSInfoDir: f.TLSInfoDir,
+		Verify:     f.Verify,
+		Requirements: []tlsmetadatainterfaces.Requirement{
+			ownership.NewOwnerRequirement(),
```

Comment:
We should have a var with Required/All list in `tlsmetadata` and refer to it here